source/SkiaSharp.Views/SkiaSharp.Views.WPF/SkiaSharp.Views.WPF.csproj

```diff
@@ -1,12 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFrameworks>net451;netcoreapp3.0</TargetFrameworks>
```

Comment:
Can we add .netstandard2.0 here? Not sure a bout the rules of a netstandard referencing a netcoreapp. Does the whole pipeline of WPF require netcoreapp at the TFM?

Reply:
I don't think so.  Even though both .NET Framework (in this case, version 4.5.1) and .NET Core 3.0+ support WPF, I don't think any version .NET Standard does.